4_advanced_usage/75_relay_servers.txt

```diff
@@ -25,134 +25,22 @@ The procedure will:
 
 ==== Installation procedure - On the relay
 
-===== Install Rudder agent
+===== Install Rudder server relay
 
-To begin, please install a regular Rudder agent on the OS.
-
-See <<install-agent, here>> for the complete reference about agent installation.
+To begin, please install a regular Rudder agent on the OS, following the
+<<install-agent, installation instructions>>, but install the 'rudder-server-relay'
```

Comment:
I find the word "but" surprising here - it leads me to think that you should not do it as indicated in the linked documentation.

How about changing "but" to "and"?